calculator_proj.rb

```diff
@@ -0,0 +1,42 @@
+puts "Welcome to the calculator"
+puts
+
+puts "The calculator can add,subtract,divide and multiply."
+
+puts "Please select your choice "
+
+choice = gets.chomp
+if choice == "add" || choice == "+"
+  puts "You chose addition"
+elsif
+  choice == "subtract" || choice == "-"
+  puts "You chose subtraction"
+elsif
+  choice == "multiply" || choice == "*"
+  puts " You chose multiplication"
+elsif
+  choice == "divide" || choice == "/"
+  puts "You chose division"
+else
+  puts " Calculator does not accept that input."
+end
+
+puts "Please give me your first number"
+first_num = gets.to_f
```

Comment:
Similarly, is there anything you could do to ensure that the user enters a number?

```ruby
num = ""
until num.to_i.to_s == num
  puts "Please enter a number"
  num = gets.chomp
end
```